### PR TITLE
hooks: Add a hook to generate Kolibri automatic_provision.json

### PR DESCRIPTION
**Pull request by dylanmccall**
https://phabricator.endlessm.com/T31679

### CODE REVIEW

File: config/defaults.ini [hunk 1]
```diff
@@ -210,6 +210,13 @@ kolibri_content_channel_eos_help = e8a879742b2249a0a4b890f9903916f7
 kolibri_content_channel_endless_key = d29581bbf4394456971b8cb807219c39
 kolibri_content_extra_channels =
 
+# Kolibri automatic provisioning configuration
+kolibri_automatic_provision_facility_name = Endless
+kolibri_automatic_provision_superusername = admin
+kolibri_automatic_provision_superuserpassword = admin
```
Comment (jprvita) [inline config/defaults.ini:216]:
```suggestion
kolibri_automatic_provision_superuser_name = admin
kolibri_automatic_provision_superuser_password = admin
```

File: hooks/image/62-kolibri-automatic-provision-install [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# images where Kolibri comes with a large selection of content and we don't
+# expect users to be interested in its other features beyond viewing
+# pre-installed content.
+
+mkdir -p "${OSTREE_VAR}"/lib/kolibri/data
+cat <<EOF > "${OSTREE_VAR}"/lib/kolibri/data/automatic_provision.json
+{
+  "facility": "${EIB_IMAGE_KOLIBRI_AUTOMATIC_PROVISION_FACILITY_NAME}",
+  "superusername": "${EIB_IMAGE_KOLIBRI_AUTOMATIC_PROVISION_SUPERUSERNAME}",
+  "superuserpassword": "${EIB_IMAGE_KOLIBRI_AUTOMATIC_PROVISION_SUPERUSERPASSWORD}",
```
Comment (jprvita) [inline hooks/image/62-kolibri-automatic-provision-install:13]:
```suggestion
  "superusername": "${EIB_IMAGE_KOLIBRI_AUTOMATIC_PROVISION_SUPERUSER_NAME}",
  "superuserpassword": "${EIB_IMAGE_KOLIBRI_AUTOMATIC_PROVISION_SUPERUSER_PASSWORD}",
```

**Review by jprvita — CHANGES_REQUESTED** [review]
Small nitpik suggestion for legibility, but otherwise LGTM (if you disagree with the suggested change, feel free to push it as-is).

**Comment by dylanmccall** [comment]
Oh, agreed, that's an ugly property name. Fixed now :) Thanks for the quick review!

**Pull request merged**